src/engraving/dom/staff.cpp

```diff
@@ -124,6 +125,71 @@ Staff* Staff::findLinkedInScore(const Score* score) const
     return nullptr;
 }
 
+std::vector<track_idx_t> Staff::getLinkedTracksInStaff(const Staff* linkedStaff, const track_idx_t originalTrack) const
+{
+    // For linked staves the length of staffList is always > 1 since the list contains the staff itself too!
+    const bool linked = staffList().size() > 1;
+
+    staff_idx_t staffIdx = linkedStaff->idx();
+
+    std::vector<track_idx_t> linkedTracks;
+    if (!linkedStaff->score()->excerpt()) {
```

Comment:
I find `score()->isMaster()` more intuitive to read for these kind of master-vs-parts checks. We can do that in a next PR